tests/specs/session-replay/payload.e2e.js

```diff
@@ -0,0 +1,78 @@
+import { config, getSR } from './helpers'
+
+export default (function () {
+  describe('Session Replay Payload Validation', () => {
+    afterEach(async () => {
+      await browser.destroyAgentSession(browser.testHandle)
+    })
+
+    it('should be gzipped', async () => {
+      await browser.url(await browser.testHandle.assetURL('rrweb-instrumented.html', config()))
+        .then(() => browser.waitForAgentLoad())
+
+      const { request: harvestContents } = await browser.testHandle.expectBlob()
+
+      expect(harvestContents.query.content_encoding).toEqual('gzip')
+      expect(harvestContents.headers['content-encoding']).toEqual('gzip')
```

Comment:
Didn't we discover that setting this header caused the browser to do pre-flight CORS calls?

Reply:
this header is only set by our testing server in the interceptor.  Should remove